packages/common-types/src/earn.ts

```diff
@@ -1,36 +1,38 @@
 import { Address } from 'viem';
-import { NetworkStruct, TokenWithIcon } from '.';
+import { NetworkStruct, TokenListId, TokenWithIcon } from '.';
 
-export type ApiStrategy = {
+export type SdkStrategy = {
```

Comment:
Can't we just use now the ones exported by the SDK?

Reply:
I just pinged out BE team, and it seems they are far from implementing this in the SDK. They suggest we replicate the types in the meantime.

Reply:
Oh that's sad, ok then